packages/flutter_tools/gradle/settings.gradle.kts

```diff
@@ -0,0 +1,7 @@
+dependencyResolutionManagement {
+  repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
```

Comment:
I think this only applies to our own plugin as defined in flutter tools and does not apply to consumers of flutter is that correct?

Reply:
yup, correct

Reply:
Perfect!